### DISCUSSION TITLE
[米家同好群]招募

### DISCUSSION THREAD

**Discussion by Soldier11-ObsidianBarracks** [share121/inter-knot#4601] [其他]
向各位米游玩家们致以敬意。
米游玩家一直希望有个好的圈内氛围已是游戏圈人尽皆知的事实。怎奈天违人愿，总有一些米哈游黑粉（以下简称“米黑”）打着以各种各样、虚实相掺的旗号，刻意扰乱米圈氛围和抹黑米游及米游玩家对外形象。对此，我们希望米游玩家团结起来，而不是被不良舆论影响，或是“胳膊肘往外拐”。
以召集米游玩家、创建一个良好氛围的圈子供各位米游玩家进行友好讨论的目的，我们一众志同道合的玩家创建了一个微信群聊，恭迎各位米游玩家通过扫描群聊二维码加入（群聊二维码见“附1”）或是添加“史蒂君ovo”好友加入（见“附2”）（或可添加附3或附4），无关人士还望止步。
本群聊为正经、合法群聊，望加入者遵守群公告中的各项规定，违者后果自负；无关人士一概不予入群，无关人士入群者核实后~~飞~~踢出群聊。
恭迎各位有意的米游玩家加入。

                 米家同好群 （writer:by史蒂君）                                 
                                             2024.9.8
附1

![image](https://github.com/user-attachments/assets/274fc6d1-3fa7-476c-bdcf-c705020db134)

附2

![image](https://github.com/user-attachments/assets/0bc493cd-fb37-43ac-947b-0d8ae24ccd71)

附3

![image](https://github.com/user-attachments/assets/7f2d5cb9-9401-41d2-aad8-c8fe0a69115f)

附4

![image](https://github.com/user-attachments/assets/0bab8c71-682e-4b65-b1f9-27d11621bb29)

（群主是附4，附3是11号，附2是本文作者）

**Comment by xiarideling** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/07/25/0302be52d1922a94378bd529cefbda6e_1187257128792751951.png" alt="诺登斯_盯" width="75" >欸嘿
